src/components/ViewRoutingSlip/RefundRequestForm.vue

```diff
@@ -26,21 +26,25 @@
         >
         </address-form>
       </v-col>
-      <v-col class="col-3 font-weight-bold"
-        :class="canEdit ? 'pt-0' : ''">
+      <v-col
+        class="col-3 font-weight-bold"
+        :class="canEdit ? 'pt-0' : ''"
+        v-if="isEditing || (!isEditing && chequeAdvice)"
```

Comment:
Hide chequeAdvice field when it's empty

Reply:
`isEditing || (!isEditing && chequeAdvice)` -> `isEditing || chequeAdvice`